### ISSUE TITLE
Add support for Color Schemes using COlor Scheme API (R2022 and newer)

### ISSUE THREAD

**Issue by enarhi** [BHoM/Revit_Toolkit#1187]
#### Description:
Now that Color Schemes are supported via the API, we should add support for them in Revit Toolkit. The extents to which we support them is TBD, but should begin with a basic pull for key graphics based workflows.

**Comment by FraserGreenroyd** [comment]
Is this not a duplicate of #1184?

**Comment by enarhi** [comment]
No, although thank you for flagging as that will be required as part of this effort. Color Schemes are a specific concept in Revit that had not previously been supported by the API, and will allow for efficiency gains once supported by us as they are heavily used by MEP and others and typically require manual (and often frustrating) input.
